docs/flow/modin/experimental/xgboost.rst

```diff
@@ -58,54 +59,49 @@ internal implementation is the following:
    ..
 
 2. On this step, the parameter `num_actors` is processed. The internal function :py:func:`~modin.experimental.xgboost.xgboost_ray._get_num_actors`
-   examines the value provided by the user and checks if it fits in the set of expected values
-   (int, "default_train", "default_predict").
+   examines the value provided by the user and checks if it fits in the set of expected values (int, None).
 
    * int - `num_actors` won't be changed. This value will be used.
-   * "default_train" - `num_actors` will be computed using condition that 1 actor should use maximum 2 CPUs.
+   * None - `num_actors` will be computed using condition that 1 actor should use maximum 2 CPUs.
```

Comment:
None should be default value (i.e. specify `num_actors=None` explicitly in the function).

Reply:
`num_actors=None` was specified for all necessary functions. Please, check again